src/intl/messages/es.json

```
@@ -292,7 +292,119 @@
+					"toYourTelegramChat": "a su Chat de Telegram como miembro",
+					"sendThisCommand": "Envíe este comando al chat con el bot:",
+					"generateToken": "Generar Token",
+					"usernameAndVerificationToken": "Nombre de Usuario y Token de Verificación:",
+					"optionalSendHelp": "(Opcional) Envíe este comando para obtener ayuda:",
+					"copyYourUsername": "Copie su nombre de usuario:",
+					"verificationToken": "Token de Verificación:",
+					"clickToGetInviteLink": "Haga clic para obtener enlace de invitación:",
+					"addToSlack": "Agregar a Slack",
+					"areYouSureRemoveBot": "¿Está seguro de que quiere eliminar el bot de Polkassembly de su chat del canal {channel}?",
+					"removingBotMeansNoNotifications": "Eliminar el bot significa no más notificaciones para el chat del canal {channel}. Manténgase conectado e informado manteniendo el bot.",
+					"preExistingSettingsWillBeChanged": "La configuración preexistente se cambiará para las siguientes redes:",
```

Biome (2.1.2)

[error] The key removingBotMeansNoNotifications was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)